src/pages/BookPage.jsx

```diff
@@ -3,34 +3,9 @@ import BookDetail from "../components/BookDetail";
 import { useEffect, useState } from "react";
 
 export default function BookPage() {
```

Comment:
Any need for this component to exist? Or could you get rid of it, and just put BookDetail directly into the route?